vite.config.ts

```diff
@@ -15,4 +13,4 @@ export default defineConfig({
       '@': fileURLToPath(new URL('./src', import.meta.url))
     }
```

Comment:
The changes appear to be minor clean-up. The removal of the comment and empty line improves code conciseness. However, removing the newline at the end of the file may cause issues with some version control systems and linters.

``` 
Add a newline at the end of the file:
})

```